params/computation_cost_params.go

```diff
@@ -168,4 +168,7 @@ const (
 	SarComputationCostIstanbul    = 834
 	XorComputationCost            = 657
 	XorComputationCostIstanbul    = 454
+
+	// computation costs for opcode added at koreCompatible Protocol Upgrade
+	RandomComputationCost = 199
```

Comment:
I am curious whether there is a reference to justify this computation cost.

Reply:
It's from benchmark test.
It will be updated.

Reply:
Reflected in `4383a1d8504f13254fd600f9feb7a3a589e130a0`
Thank you for your reveiw.